script.js

```diff
@@ -1,105 +1,305 @@
         return "paper";
 
     } else {
+        computerRockPic.style.transform = "scale(.5)";
+        computerPaperPic.style.transform = "scale(.5)";
+        computerScissorsPic.style.transform = "scale(1.5)";
         return "scissors";
-    }
+    } 
 }
 
-//console.log(getComputerChoice(3));
+//create playRound function
 
-//function to get human choice with prompt
-// function to get human choice
-function getHumanChoice() {
-    let promptChoice = prompt("Rock, paper, or scissors?");
+function playRound(humanChoice, computerChoice) {
 
-        if(promptChoice.toLowerCase() === "rock") {
-            return "rock";
+    if (humanChoice === computerChoice) { 
+        winner.textContent = `It's a draw. Play again.`;
+    }
 
-        } else if(promptChoice.toLowerCase() === "paper") {
-            return "paper" 
+    else if (humanChoice === 'rock' && computerChoice === 'scissors' || humanChoice === 'paper' && computerChoice === 'rock' || humanChoice === 'scissors' && computerChoice === 'paper') {
+        winner.textContent = `${humanChoice} beats ${computerChoice}. You win!`;
+        humanScore+= 1;
+    }   
 
-        } else if(promptChoice.toLowerCase() === "scissors") {
-            return "scissors";
-        
-        } else {
-            console.log("Invalid choice. Please try again.");
-        }
-}
+    else { 
+        winner.textContent = `${computerChoice} beats ${humanChoice}. You lose.`;
+        computerScore+= 1;
+    }
 
-//console.log(getHumanChoice());
+} 
 
-const humanSelection = getHumanChoice();
-const computerSelection = getComputerChoice(3);
+function playGame() {
 
-//function for 5 round and end of game
+    if ((humanScore < 5) || (computerScore < 5)) {
 
-function playGame() {
-    //declare score variables and round counter
-    
-    let humanScore = 0;
-    let computerScore = 0;
-    let round = 0;
-// function for one round 
-
-    function playRound(humanChoice, computerChoice) {
-
-        if (humanChoice === computerChoice) {
-            console.log(`Round: ${round}`);
-            console.log(`It's a draw. Play again.`);
-        }
-
-        else if (humanChoice === 'rock' && computerChoice === 'scissors' || humanChoice === 'paper' && computerChoice === 'rock' || humanChoice === 'scissors' && computerChoice === 'paper') {
-            console.log(`Round: ${round}`);
-            console.log(`${humanChoice} beats ${computerChoice}. You win!`);
-        return humanScore++;
-        }   
-    
-        else { 
-            console.log(`Round: ${round}`);
-            console.log(`${computerChoice} beats ${humanChoice}. You lose.`);
-        return computerScore++;
-        }
-    } 
+        rock.addEventListener('click', () => {
+            rock.style.transform = "scale(1.5)";
+            paper.style.transform = "scale(.5)";
+            scissors.style.transform = "scale(.5)";     
+            let computerChoice = getComputerChoice(3);
```

Comment:
`getComputerChoice` does not accept arguments based on the way you defined it above. therefore, you do not need to include the 3 in there. it may be causing issues or failing to run properly